### ISSUE TITLE
SchemaGen ignores namespace and file name

### ISSUE THREAD

**Issue by chrouknorris** [IntershopCommunicationsAG/jaxb-gradle-plugin#10]
SchemaGen tasks ignores the  namespace Schema and file name, so it's all time generate one fine schema1.xsd, also when we have a multiple configuration. 

Can you help me with the issue ?

**Comment by m-raab** [comment]
Can you send me an example?
This is the parameter for the used ant task
`            getNamespaceConfigs().each { namespace, fileName ->
                schema(namespace: namespace, file: fileName)
            }
`
But unfortunately, there is no test?

**Comment by chrouknorris** [comment]
Yes I said this, but unfortunateley same it's doesn't take in consideration by ant schemagen task. 

You can just add a namespaceconfigs in this test :
 https://github.com/IntershopCommunicationsAG/jaxb-gradle-plugin/blob/01791202ace56b1e753a5860188900047bba2dd9/src/test/groovy/com/intershop/gradle/jaxb/SamplesSpec.groovy#L737
for example :
` namespaceconfigs = [ 'http://www.corporate.com/xml/ns/corporate/feature/1.0' : 'feature.xsd' ]`
and change the line to : 
`File schemaFile = new File(testProjectDir, 'build/generated/jaxb/schema/test/feature.xsd')`

Expected : test ok with file schema feature.xsd 
Obtain : test faild

**Comment by m-raab** [comment]
I found something ....

https://www.ibm.com/support/knowledgecenter/en/SSEQTP_8.0.0/com.ibm.websphere.base.doc/info/aes/ae/twbs_jaxbjava2schema.html

**Comment by chrouknorris** [comment]
I deepened my research, the configuration is well setter in shcemaGenTask. but not taken into account for the generation.

We can use what is described in this link to manage namespace, but not the file name of xsd.
